plc4go/internal/opcua/SecureChannel.go

```diff
@@ -1416,13 +1438,13 @@ func (s *SecureChannel) keepAlive() {
 							s.log.Debug().Type("type", messagePDU).Msg("Not relevant")
 							return false
 						}
-						return openResponse.GetRequestId() == transactionId
+						return openResponse.GetMessage().GetSequenceHeader().GetRequestId() == transactionId
 					},
 					func(message spi.Message) error {
 						opcuaAPU := message.(readWriteModel.OpcuaAPU)
 						messagePDU := opcuaAPU.GetMessage()
 						opcuaOpenResponse := messagePDU.(readWriteModel.OpcuaOpenResponse)
-						readBuffer := utils.NewReadBufferByteBased(opcuaOpenResponse.GetMessage(), utils.WithByteOrderForReadBufferByteBased(binary.LittleEndian))
+						readBuffer := utils.NewReadBufferByteBased(opcuaOpenResponse.(readWriteModel.BinaryPayloadExactly).GetPayload(), utils.WithByteOrderForReadBufferByteBased(binary.LittleEndian))
```

Comment:
is it ensured that this is always `readWriteModel.BinaryPayloadExactly`?. Otherwise this part will panic